content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -160,29 +160,30 @@ There are some implicit conventions worth noting here:
-    2. the incoming Pod has no chances to be scheduled onto this kind of nodes - in the above example, suppose a "node5" carrying label `{zone-typo: zoneC}` joins the cluster, it will be bypassed due to the absence of label key "zone".
+
+  1. the Pods located on those nodes do not impact `maxSkew` calculation - in the above example, suppose "node1" does not have label "zone", then the 2 Pods will be disregarded, hence the incomingPod will be scheduled into "zoneA".
+  2. the incoming Pod has no chances to be scheduled onto this kind of nodes - in the above example, suppose a "node5" carrying label `{zone-typo: zoneC}` joins the cluster, it will be bypassed due to the absence of label key "zone".
 
 - Be aware of what will happen if the incomingPod’s `topologySpreadConstraints[*].labelSelector` doesn’t match its own labels. In the above example, if we remove the incoming Pod’s labels, it can still be placed onto "zoneB" since the constraints are still satisfied. However, after the placement, the degree of imbalance of the cluster remains unchanged - it’s still zoneA having 2 Pods which hold label {foo:bar}, and zoneB having 1 Pod which holds label {foo:bar}. So if this is not what you expect, we recommend the workload’s `topologySpreadConstraints[*].labelSelector` to match its own labels.
 
 - If the incoming Pod has `spec.nodeSelector` or `spec.affinity.nodeAffinity` defined, nodes not matching them will be bypassed.
 
-    Suppose you have a 5-node cluster ranging from zoneA to zoneC:
+  Suppose you have a 5-node cluster ranging from zoneA to zoneC:
+
+  ```
+  +---------------+---------------+-------+
+  |     zoneA     |     zoneB     | zoneC |
+  +-------+-------+-------+-------+-------+
+  | node1 | node2 | node3 | node4 | node5 |
+  +-------+-------+-------+-------+-------+
+  |   P   |   P   |   P   |       |       |
+  +-------+-------+-------+-------+-------+
+  ```
 
-    ```
-    +---------------+---------------+-------+
-    |     zoneA     |     zoneB     | zoneC |
-    +-------+-------+-------+-------+-------+
-    | node1 | node2 | node3 | node4 | node5 |
-    +-------+-------+-------+-------+-------+
-    |   P   |   P   |   P   |       |       |
-    +-------+-------+-------+-------+-------+
-    ```
+  and you know that "zoneC" must be excluded. In this case, you can compose the yaml as below, so that "mypod" will be placed onto "zoneB" instead of "zoneC". Similarly `spec.nodeSelector` is also respected.
 
-    and you know that "zoneC" must be excluded. In this case, you can compose the yaml as below, so that "mypod" will be placed onto "zoneB" instead of "zoneC". Similarly `spec.nodeSelector` is also respected.
+  {{< codenew file="pods/topology-spread-constraints/one-constraint-with-nodeaffinity.yaml" >}}
 
-    {{< codenew file="pods/topology-spread-constraints/one-constraint-with-nodeaffinity.yaml" >}}
-    
```

Comment:
line 185 should not be changed either.

Reply:
It's just lints of vscode lint plugin which is being launched uponsave. I didn't change anything

Reply:
if you compare the preview page to the current unchanged version, you will find out whether this lint is doing things right.

Reply:
I see what you mean. This is weird because in GITHUB preview this looks fine.
Would you want me to edit this without any linting and just replace the broken link?
If so, in a new PR for a "clean" commit tree, or by updating this branch?

Reply:
Just updating this branch should work. You can decide whether to keep the good linting.